prime-react/src/main/scala/react/primereact/AccordionMultiple.scala

```diff
@@ -31,6 +31,10 @@ case class AccordionMultiple(
   def withMods(mods:          TagMod*)     = addModifiers(mods)
 
   override val rawActiveIndex = activeIndices.map(_.map(_.toDouble).toJSArray)
+  override val rawOnTabChange =
+    onTabChange.map(
+      _.compose[Double | js.Array[Double]](_.asInstanceOf[js.Array[Int]].toList)
```

Comment:
Did I know you could just `asInstanceOf` an `Array[Double]` to an `Array[Int]`? If not, I should remember it.